modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="cco-ccoctl-creating-at-once_{context}"]
+= Creating AWS resources with a single command
+
+If you do not need to review the JSON files that the `ccoctl` creates before modifying AWS resources, and if the process the `ccoctl` uses to create AWS resources automatically meets the requirements of your organization, you can use the `ccoctl aws create-all` command to automate the creation of AWS resources.
+
+Otherwise, you can create the AWS resources individually.
+
+//to-do if possible: xref to modules/cco-ccoctl-creating-individually.adoc for `create the AWS resources individually`
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `_<path_to_ccoctl_output_dir>_` to refer to this location.
+====
+
+.Prerequisites
+
+* Extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. Extract the list of `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc adm release extract --credentials-requests --cloud=aws --to=__<path_to_directory_with_list_of_credentials_requests>__/credrequests quay.io/__<path_to>__/openshift-release:__<version>__
+----
+
+. Process all `CredentialsRequest` objects in the `credrequests` directory with `ccoctl`:
++
+[source,terminal,subs="+quotes"]
+----
+$ ccoctl aws create-all --name=__<name>__ --region=__<aws_region>__ --credentials-requests-dir=__<path_to_directory_with_list_of_credentials_requests>__/credrequests
+----
+
+.Verification
+
+You can verify that the {product-title} secrets are created by checking the secret files in the `_<path_to_ccoctl_output_dir>_/manifests` directory.
```

Comment:
Not sure if the same rules apply for `Verification` as they do for `Procedure`, but usually we suggest that an intro sentence comes before the header. So after the heading, it's just a list of ordered steps. In this case, it doesn't make sense to have an intro sentence before `Verification` because it's a sub-part of the Procedure. `¯\_(ツ)_/¯`

One solution is to eliminate this intro sentence and in the step, write:

> List the secret files in the `<path_to_ccoctl_output_dir>/manifests` directory to verify that the {product-title} secrets were created.

Reply:
That's actually a good point, especially since the intro and the step are fairly similar. Thanks!